src/index.js

```diff
@@ -73,4 +74,26 @@ export default class Banana {
     }
     return messageKey
   }
+
+  /**
+   * Register a plugin for the library's message parser
+   * Example:
+   * <pre>
+   *   banana.registerParserPlugin('sysop', nodes => {
+   *     return mw.config.get('wgUserGroups').includes('sysop') ?
```

Comment:
Since this library is mediawiki independent, could you please change the above comment a little bit more generic? Just avoid mw variable and use some plain javascript conditions.

Wanted to make sure developers working on non-mediawiki getting confused by this globals

Reply:
Ok, switched it to the foobar example (as in this PR description)